Sources/VaporToolbox/Version.swift

```diff
@@ -22,7 +22,10 @@ public final class Version: Command {
         console.success("\(version)")
 
         guard verifySwiftProject() else { return }
-        guard try verifyVaporProject() else { return }
+        guard try isVaporProject(with: console) else {
+            console.warning("No Vapor dependency detected, unable to log Framework Version")
```

Comment:
i've seen this warning confuse people alot. we should probably just not log the vapor framework version if we're not inside a repo